Components/List/index.css

```diff
@@ -0,0 +1,16 @@
+.kash-basic-ol {
+  list-style: none;
+}
+.kash-basic {
+  background-color: var(--white);
+  list-style: none;
+  box-shadow: 0.1rem 0.1rem 0.1rem 0.1rem var(--gray);
```

Comment:
Better to use pixel values since they are very small

Reply:
sure thanks